src/components/TodosList.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉